gax-java/gax/src/main/java/com/google/api/gax/core/GaxProperties.java

```diff
@@ -148,4 +150,31 @@ static Optional<String> getBundleVersion(Class<?> clazz) {
       return Optional.empty();
     }
   }
+
+  /**
+   * Returns the Protobuf runtime version as reported by com.google.protobuf.RuntimeVersion,
+   * if class is available, otherwise by reading from MANIFEST file. If niether option is available defaults to
+   * protobuf version 3 as RuntimeVersion class is available in protobuf version 4+
+   */
+  @VisibleForTesting
+  static String getProtobufVersion(IClassLoaderWrapper classLoader, Class clazz) {
```

Comment:
Please also make sure to rename this classloader argument, as this is not a classloader.